### DISCUSSION TITLE
Eassy access "Lock" device not supported?

### DISCUSSION THREAD

**Discussion by sonybansi** [Koenkk/zigbee2mqtt#8056] [Devices]
EasyAccess EasyCode903G2.1 is a supported device in Zigbee2mqtt, but in my HA setup show unsupported device. 

```
Zigbee2MQTT:warn  2021-07-15 19:51:50: Received message from unsupported device with Zigbee model 'EasyCode903G2.1' and manufacturer name 'Datek Wireless'
Zigbee2MQTT:warn  2021-07-15 19:51:50: Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.
Zigbee2MQTT:info  2021-07-15 19:24:02: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"<0x000d6f000c71f2cb>"},"type":"entity_not_found"}'
Zigbee2MQTT:error 2021-07-15 19:24:02: Entity '<0x000d6f000c71f2cb>' is unknown
Zigbee2MQTT:info  2021-07-15 19:01:38: 0x000d6f000c71f2cb (0x000d6f000c71f2cb): Not supported (EndDevice)
```
![Screenshot_20210715_195853](https://user-images.githubusercontent.com/28208396/125835360-ca98f88e-c038-43b8-9322-4a3faac19de6.jpg)

**Comment by sjorge** [comment]
Can you post the database entry for this device, maybe it's a slightly newer or older version.

**Reply by sonybansi** [reply]
Where can I find database entry? I have pictures of device. 
![IMG_20210303_104636](https://user-images.githubusercontent.com/28208396/126019381-d3079712-4788-4bca-8fd9-2181432a3bb7.jpg)
![IMG_20210303_104252](https://user-images.githubusercontent.com/28208396/126019384-45031c73-4ed0-4a1d-8020-c69b6a2b98e7.jpg)
